integrations/terraform/Makefile

```diff
@@ -29,7 +29,8 @@ clean: tfclean
 
 .PHONY: build
 build: clean
-	GOOS=$(OS) GOARCH=$(ARCH) $(CGOFLAG) go build -o $(BUILDDIR)/terraform-provider-teleport $(BUILDFLAGS)
+# Turning off GOWORK to prevent missing package errors.
+	GOWORK=off GOOS=$(OS) GOARCH=$(ARCH) $(CGOFLAG) go build -o $(BUILDDIR)/terraform-provider-teleport $(BUILDFLAGS)
```

Comment:
We should probably GOWORK=off in the bin/tfplugindocs target below.

Other targets like `test` and `gen-tfschema` also have potentially problematic Go invocations, but we can wait for them to break.

Another alternative is to pass GOWORK=off straight from the `terraform-resources-up-to-date`. Or, do nothing of this and set it straight in the make invocation in the shell (so no code changes).

Reply:
Added to the `bin/tfplugindocs` target.